src/web/resources/report.js

```diff
@@ -265,6 +266,157 @@ const ClientGraph = new Component('#graphs', {
     }
 })
 
+const MergedCostAccuracy = new Component('#merged-cost-accuracy', {
+    setup: async () => {
+        console.log("make cost accuracy");
+        const points_json = await (async () => {
+            const get_points_store = {}
+            
+            const get_points_memo = async () => {
+                if (get_points_store.value) { return get_points_store.value }
+                const ps = await get_json('cost-accuracy.json');
+                get_points_store.value = ps;
+                return get_points_store.value;
+            }
+            const get_json = url => fetch(url, {
+                // body: `_body_`,
+                headers: {"content-type": "text/plain"},
+                method: "GET",
+                mode: 'cors'
+                }).then(async response => {
+                //await new Promise(r => setTimeout(() => r(), 200) )  // model network delay
+                return await response.json()
+            })
+            return get_points_memo()
+        })()
+
+        const plot = async () => {
+            // NOTE ticks and splitpoints include all vars, so we must index
+            const { best, first, points } = points_json;
+
+            const out = Plot.plot({
+                marks: [
+                    Plot.line(points, {x: d => d[0], y: d => d[1], stroke: "red"}),
+                    Plot.dot(first, {x: first[0], y: first[1], fill: "black"})
+                ],
+                grid: true,
+                width: '800',
+                height: '400',                
+                    x: {
+                        label: `Cost`,
+                        domain: [0, best[0]]
+                    },
+                    y: {
+                        label: "Sum of error bits",
+                        domain: [0, best[1]],
+                    },
+            })
+            out.setAttribute('viewBox', '0 0 800 460')
+            return out
+        }
+        function html(string) {
+            const t = document.createElement('template');
+            t.innerHTML = string;
+            return t.content;
+        }
+        async function render() {
+            const options_view = html(`
```

Comment:
I think you don't need lines 323 down to line 337 here since you're not doing anything with showing different variables and functions. Can also get rid of the html() function on line 317-321 unless it's being used for templating somewhere I missed.

Reply:
(Also would get rid of options_view reference in line 338)

Reply:
Well, maybe at some point we will want plot options, though, so it could be useful to leave it in there. So I would actually just get rid of the contents of the plot options div and the onclick handlers below that were binding to the contents. ie lines 325-328 and 331-337.